test/integration/controllers/category.spec.js

```diff
@@ -12,12 +12,12 @@ let accessToken
 
 let categoryBody = {
   name: 'Languages',
-  categoryIcon: { path: 'mocked-path-to-icon', color: '#66C42C' }
+  categoryIcon: { icon: 'mocked-path-to-icon', color: '#66C42C' }
```

Comment:
change categoryIcon to appearance here too

Reply:
fixed